src/get-started/npm-script.md

```diff
@@ -14,6 +14,9 @@ cli` to generate a service worker.
     npm install --save-dev workbox-cli
     ```
 
+    **Note:** You can skip this step if you have already installed
```

Comment:
Can you leave this out? Keeping things consistent—always ensuring that there's a local copy of `workbox-cli`—means that builds will work regardless of the computer's global state. Moreover, the version of `workbox-sw` that gets used in the generated SW file is tied to the version of `workbox-cli` that's being used, so giving folks control over that versioning via their local `package.json` is important.